modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -44,7 +233,51 @@ Returns one of the following strings, based on the value of expression:
 * "object"
 * "binary"
 
+== TYPE(expression)
+
+=== Description
```

Comment:
Needs separate Description, Arguments, and Return Value sections, as per functions template

```suggestion
=== Description

Checks the type of the supplied expression.

=== Arguments

expression:: [Required] The expression to check.

=== Return Value
```